package.json

```diff
@@ -21,8 +21,10 @@
     "serve-storybook": "npx serve storybook-static -p 8080"
   },
   "dependencies": {
+    "storybook": "^9.1.10",
     "@mantine/core": "8.2.1",
     "@mantine/hooks": "8.2.1",
+    "@storybook/test": "^8.6.14",
```

Comment:
Those should be dev dependencies

Reply:
I moved the dependencies to dev via [commit](https://github.com/Deadlink-Hunter/Broken-Link-Website/pull/156/commits/184704aafc6b33b34dbbd4660977fe0f04d2d4c5)